core/src/main/java/com/rudderstack/android/sdk/core/EventRepository.java

```diff
@@ -40,15 +42,21 @@
     private final List<RudderMessage> eventReplayMessageQueue = Collections.synchronizedList(new ArrayList<RudderMessage>());
     private String authHeaderString;
     private String anonymousIdHeaderString;
-    private int versionCode;
+    private Context context;
     private RudderConfig config;
     private DBPersistentManager dbManager;
     private RudderServerConfigManager configManager;
     private RudderPreferenceManager preferenceManager;
     private RudderEventFilteringPlugin rudderEventFilteringPlugin;
+    private RudderFlushWorkManager rudderFlushWorkManager;
```

Comment:
This might crash if work manager is not implemented by the user

Reply:
Yeah, so to crash guard this we are already checking if the WorkManager is being present on the Classpath or not over [here](https://github.com/rudderlabs/rudder-sdk-android/blob/2464a58d22db8f81f5256d4ad858fa4a2576cad9/core/src/main/java/com/rudderstack/android/sdk/core/RudderFlushWorkManager.java#L66)